### DISCUSSION TITLE
Alignment Considerations for hse_kvs_put and hse_kvs_get

### DISCUSSION THREAD

**Discussion by victorstewart** [hse-project/hse#20] [Q&A]
do `hse_kvs_put` and `hse_kvs_get` either expect or benefit from the memory locations pointed to by their key / value inputs 8 byte aligned, or does it not matter?

**Comment by tristan957** [comment]
As far as expecting aligned memory, the answer is no.

As for any chance at increased performance, I will get some answers and get back to you, or direct someone with answers here. Thanks!
